ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -27,6 +27,10 @@ let with_lock = Xapi_stdext_threads.Threadext.Mutex.execute
 let lwsmd_service = "lwsmd"
 
 module Lwsmd = struct
+  let last_restart = Mtime_clock.now () |> ref
+
+  let restart_interval = 3600.
```

Comment:
This should be an `Mtime.Span.t` value:

```
let restart_interval = Mtime.(1 * Span.hour)
```

or similar - depending on what span you want.

Reply:
well, we could do this, 
However, I do not find Span.hour in https://v3.ocaml.org/p/mtime/1.2.0/doc/Mtime/Span/index.html, 
maybe `let restart_interval = Mtime.(3600* 1000000000 * Span.one)` ?

Reply:
transformer used `1.2.0`, 
`mtime                   1.2.0       Monotonic wall-clock time for OCaml`, 
so I will not use `Mtime.Span.(1 * hour)` as this requires backport.

Reply:
I upgraded my Mtime and it worked 
```
utop # let restart_internval = Mtime.Span.hour;;
val restart_internval : Mtime.span = <abstr>
```